modules/service/src/main/scala/lucuma/itc/service/ItcMappings.scala

```diff
@@ -376,7 +376,7 @@ object ItcMapping extends Encoders {
               val units = unitsItems.get(u)
               (band, value, units)
                 .mapN { (b, v, u) =>
-                  b -> u.withValueTagged(BrightnessValue(v.toInt * 1000))
+                  b -> u.withValueTagged(BrightnessValue(v.toInt))
```

Comment:
Won't this truncate the brightness to an integer?  Should it be `BrightnessValue.fromBigDecimal.get(v)`?